modules/add.js

```diff
@@ -0,0 +1,26 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable class-methods-use-this */
+import { Storage } from './storage.js';
+
+const storage = new Storage();
+
+export class Add {
+  constructor() {
+    return null;
+  }
+
+  addBook(book = null, author = null) {
+    if (book && author) {
+      const newBook = {
+        book,
+        author,
+      };
```

Comment:
- [ ] [OPTIONAL] it would be better if you use es6 `arrow function` instead of `normal method` in order to practice working with both